Update Welcome Screen links

conath opened the pull request:
Update User Guide link on iOS
Add Support link on macOS and FAQ link on iOS

and remove some unnecessary `Spacer()`s

Only thing missing is the localization.

osy commented:
1. I think the macOS guide applies to iOS 14+ version as well. (at least it's better than the iOS 13 guide which is useless and is why I didn't link to it. I just never had time to write the v2 guide).
2. Similarly the support page for macOS also applies.

I think therefore it may make sense to just link to the macOS page even for iOS. Unless you read the address bar, it won't be too much of an issue. Maybe at some point I'll unify the two sites.

conath commented:
I have updated it to include your feedback. It looks like this now on all platforms:
<img width="430" alt="Bildschirmfoto 2021-05-09 um 21 26 23" src="https://user-images.githubusercontent.com/12073163/117584498-a96b3a00-b10d-11eb-87d0-727c9f63aaf7.png">

Notice that I changed the gallery icon to match the circular icon in the other buttons. 

I've cancelled the CI run for the last push because it's such a minor change.
